lib/ansible/galaxy/collection.py

```diff
@@ -1216,3 +1422,13 @@ def _consume_file(read_from, write_to=None):
         data = read_from.read(bufsize)
 
     return sha256_digest.hexdigest()
+
+
+def get_galaxy_metadata_path(b_path):
+    b_default_path = os.path.join(b_path, b'galaxy.yml')
+    candidate_names = [b'galaxy.yml', b'galaxy.yaml']
```

Comment:
You have only one variant in many places hardcoded. I'm expecting this to cause bugs... How about having at least a module-global constant for these? Or maybe a structure (namedtuple?) that holds both filename and its contents.

Reply:
@webknjaz Where is that? Can you give me an example? We've only supported `galaxy.yml` in the past. If this looks invasive I'll just remove this supporting both for now, and it can be addressed in a follow-up PR.

Reply:
Line 1211

Reply:
Fixed, thanks